state_machine/drone.py

```diff
@@ -72,6 +73,33 @@ def __init__(self, address: str = "", baud: int | None = None) -> None:
         with open("flight/data/attempted_drops.json", "w", encoding="utf8") as file:
             json.dump({}, file)
 
+    async def _send_servo_msg(self, servo_num: int, pwm: int) -> None:
+        """Send a DO_SET_SERVO MAVLink message to the drone.
+
+        Parameters
+        ----------
+        servo_num : int
+            The number of the servo to control.
+            This should be the same value that is shown in MissionPlanner
+            and the parameters of the drone.
+        pwm : int
+            The PWM value to send to the servo.
+        """
+        msg = self.vehicle.message_factory.command_long_encode(
```

Comment:
Could do a type hint here. But I don't think it is necessary since it is just this place and it is just being made into a mavlink message.